snowflake_queries/snowflake_key_user_password_login_query.yml

```diff
@@ -0,0 +1,27 @@
+AnalysisType: scheduled_query
+QueryName: Query.Snowflake.KeyUserPasswordLogin
+Enabled: false
+Description: >
+  Detects when a user with a configured RSA key logs in with a password
+AthenaQuery: >
+  /* athena query not supported */
+  SELECT count(1)
+SnowflakeQuery: >
+  --return instances where a user who has key-based login configured logs in with a password
+  --this was adapted from a Security Feature Checklist query
+
+   SELECT
+     u.name,
+     first_authentication_factor,
+     second_authentication_factor,
+     count(*)
+  FROM snowflake.account_usage.login_history as l
+  JOIN snowflake.account_usage.users u on l.user_name = u.name and has_rsa_public_key = 'true'
```

Comment:
is this searching all time? should we scope it to look back at the same timeframe as the schedule? (24 hours?)